presto-sheets/src/test/java/io/prestosql/plugin/sheets/SheetsSimpleIntegrationTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.sheets;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.Session;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tests.AbstractTestQueryFramework;
+import io.prestosql.tests.DistributedQueryRunner;
+import org.testng.annotations.Test;
+
+import static io.prestosql.plugin.sheets.TestSheetsPlugin.TEST_METADATA_SHEET_ID;
+import static io.prestosql.plugin.sheets.TestSheetsPlugin.getTestCredentialsPath;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+
+public class SheetsSimpleIntegrationTest
```

Comment:
Please rename to `TestGoogleSheets`. Convention in Presto is to prefix test class names from `Test`